src/directory_schema/cli.py

```diff
@@ -0,0 +1,39 @@
+import argparse
+import sys
+import os
+
+from jsonschema.exceptions import SchemaError
+from yaml import safe_load as load_yaml
+
+from directory_schema.directory_schema import validate_dir, DirectoryValidationErrors
+
+
+def dir_path(string):
+    if os.path.isdir(string):
+        return string
+    else:
+        raise Exception(f'"{string}" is not a directory')
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('dir', metavar='DIRECTORY', type=dir_path,
+                        help='Directory to validate')
+    parser.add_argument('schema', metavar='SCHEMA', type=argparse.FileType('r'),
+                        help='Schema (JSON or YAML) to validate against')
```

Comment:
Just a heads up - I am not picky about this but you use only "JSON Schema" a lot when that could also include `yaml` (like on https://github.com/hubmapconsortium/directory-schema/pull/2/files#diff-6caf4c857352f867df6b48c6622fcf38R34).

Reply:
hmm: "JSON Schema" is a thing, but a particular schema instance can be written in YAML (because YAML is a superset of JSON), and could even be validating a YAML document, but it would still be **JSON** Schema. Is that persuasive?